cloud/pkg/cloudhub/servers/httpserver/nodetask/checknodename.go

```diff
@@ -0,0 +1,43 @@
+package nodetask
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/emicklei/go-restful"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog/v2"
+
+	"github.com/kubeedge/kubeedge/cloud/pkg/common/client"
+)
+
+func CheckNodeName(request *restful.Request, response *restful.Response) {
+	// get nodeName
+	nodeName := request.QueryParameter("nodeName")
+	if nodeName == "" {
+		err := response.WriteErrorString(http.StatusBadRequest, "nodeName parameter is required")
+		if err != nil {
+			klog.Errorf("failed to send the task resp to edge , err: %v", err)
+		}
+		return
+	}
+
+	_, err := client.GetKubeClient().CoreV1().Nodes().Get(context.TODO(), nodeName, metav1.GetOptions{})
+	if err != nil {
```

Comment:
This API is used to check whether the node name conflicts. If the node is not found, 200 should be returned, and 409 should be returned if the node is found.

Reply:
Thank you for your suggestion, but I think the original meaning should be followed on the server side, and if the returned status code is used, it should be controlled by the client logic, because the interface is not necessarily used for node name checking later, it may be directly used to query nodes, so the return of the original status code will be easier to understand.

Reply:
In that case you need to change the service name. The meaning of checking the node name here is to check whether the node name conflicts...

Reply:
Thank you for your suggestion. I have made corresponding modifications.